### DISCUSSION TITLE
The bottom sheet closes when using a color picker.

### DISCUSSION THREAD

**Discussion by GaylordP** [gorhom/react-native-bottom-sheet#2079] [Q&A]
Hello and thank you for your library and the effort you put into it :)

I have a specific case involving a Color Picker inside a Bottom Sheet. Unfortunately, as soon as I lift my finger (to select a color), the Bottom Sheet closes:

https://github.com/user-attachments/assets/06f75561-11fc-4967-bdd6-ede8853934d5

I’ve tried using [disallowInterruption](https://gorhom.dev/react-native-bottom-sheet/troubleshooting#my-component-gesture-interaction-gets-conflicted-with-bottom-sheet-interactions-), [enableContentPanningGesture, or enablePanDownToClose,](https://gorhom.dev/react-native-bottom-sheet/props#enablecontentpanninggesture) but without success :/

```
      ...
      <BottomSheet
        ref={bottomSheetColorPickerRef}
        title={t('label.colorSelector')}>
        {currentColorPicker && (
          <ColorPicker
            value={localInterface.colors[currentColorPicker]}
            onComplete={returnedResults => {
              onUpdateColor(currentColorPicker, returnedResults.hex);

              onCloseBottomSheetColorPicker();
            }}>
            <FormContainer>
              <Preview />
              <Panel1 />
              <HueSlider />
            </FormContainer>
          </ColorPicker>
        )}
      </BottomSheet>
      ....
```

I understand that it’s challenging for you to provide a solution since this involves interaction between two libraries, but if you have any ideas, I’d greatly appreciate it :)

Thanks :)

**Comment by OtayNacef** [comment]
Provide more details, the library used for the ColorPicker and  versions.
Also try wrapping it with BottomSheetView
